packages/network-controller/tests/NetworkController.test.ts

```diff
@@ -5896,7 +4784,7 @@ async function waitForPublishedEvents<E extends NetworkControllerEvents>(
 
       const stopTimer = () => {
         if (timer) {
-          clearTimeout(timer);
```

Comment:
This is now printing a warning, probably due to the upgrade to Jest 27. I never noticed before that at this point in time, this is the mocked version of `clearTimeout` that comes from Jest's fake timers and not the original version.